pkg/redshift

```diff
@@ -0,0 +1,23 @@
+[mirrors]
+https://github.com/jonls/redshift/archive/b210813c7ffdd0eae75eb60c7bbbb62b6b109ebe.tar.gz
+
+[vars]
+tarball=redshift-git-b210813.tar.gz
+tardir=redshift-b210813c7ffdd0eae75eb60c7bbbb62b6b109ebe
+filesize=639073
+sha512=46c49a385ed0f05a2c0f2dd359d728728c8b21f5d40cf0da8820da5c2d8711ae5a562e126a3c8604c3f15705f08973251c277dc0261ad9fc1150b169b1e7b7be
+pkgver=1
+
+[deps]
+intltool
+
+[build]
+
+echo 'prefix="$butch_prefix"' > config.mak
```

Comment:
does that still work ? since its broken due to your quoting
if it still works, then that line can probably be removed entirely.
if not, then it needs to be `echo "prefix=$butch_prefix" > config.mak`

Reply:
Not sure if that would break it or not, but I just commented the line entirely and everything still ended up in /opt. I guess its not needed.

Fixed.